libs/sdk-embedding/src/iframe/ad.ts

```diff
@@ -779,6 +784,11 @@ export namespace EmbeddedAnalyticalDesigner {
          * The minimal opened insight information
          */
         insight: IObjectMeta;
+
+        /**
+         * Definition of insight
+         */
+        definition: GdcVisualizationObject.IVisualizationObjectContent;
```

Comment:
I propose sending IInsightDefinition. it is the platform agnostic representation of the insight (realized by vis object).

while we use bear-specific stuff on the existing post messages - that is purely because the way the messages were done way back.